.github/workflows/main.yaml

```diff
@@ -62,3 +62,24 @@ jobs:
           artifacts: "${{ steps.check-release.outputs.artifacts }}"
           bodyFile: "CHANGELOG.md"
           token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish to gitlab
```

Comment:
Is it also possible to use maven config with profiles maybe? :thinking:

Reply:
Yes but figuring out how to handle the maven setting and replacing the server-credentials in a save way would have taken more time and probable a lot more of code.